pages/api/users/index.ts

```diff
@@ -30,6 +30,18 @@ const userHandler: NextApiHandler = async (req: NextApiRequest, res: NextApiResp
       }
     }
 
+    case "GET": {
+      let role: Roles | undefined = undefined;
+      if (req.query && req.query.role) role = req.query.role as Roles;
```

Comment:
Should the GET route return 400 bad request error if the "role" query param is invalid (i.e. not one of Admin/Volunteer/Student/Staff/Parent)? Currently it returns an empty array on an invalid query param with status code 200